aspnetcore/tutorials/your-first-mac-aspnet.md

```diff
@@ -1,81 +1,71 @@
 ---
 title: Build an ASP.NET Core app on a Mac Using Visual Studio Code | Microsoft Docs
-author: ardalis
-description: This article will walk you through creating your first web application on a Mac using the Yeoman generator for ASP.NET Core and Visual Studio Code
+author: spboyer
+description: This article will walk you through creating your first web application on a Mac using the dotnet CLI for ASP.NET Core and Visual Studio Code
 keywords: ASP.NET Core, macOS, Yeoman, generator-aspnet, Visual Studio Code
 ms.author: riande
 manager: wpickett
-ms.date: 10/14/2016
+ms.date: 02/22/2017
 ms.topic: article
 ms.assetid: dcc08e09-e73e-4feb-84ce-8219b7e544ef
 ms.technology: aspnet
 ms.prod: aspnet-core
 uid: tutorials/your-first-mac-aspnet
 ---
-# Your First ASP.NET Core Application on a Mac Using Visual Studio Code
-
-By [Daniel Roth](https://github.com/danroth27), [Steve Smith](http://ardalis.com), [Rick Anderson](https://twitter.com/RickAndMSFT), and [Shayne Boyer](https://twitter.com/spboyer)
+# Your first ASP.NET Core application on a Mac using Visual Studio Code
 
 This article will show you how to write your first ASP.NET Core application on a Mac.
 
-## Setting Up Your Development Environment
-
-To setup your development machine download and install [.NET Core](https://microsoft.com/net/core) and [Visual Studio Code](https://code.visualstudio.com) with the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp). Node.js and npm is also required. If not already installed visit [nodejs.org](https://nodejs.org/en/download/package-manager/#osx).
+## Setting up your development environment
 
-## Scaffolding Applications Using Yeoman
+To setup your development machine download and install [.NET Core](https://microsoft.com/net/core) and [Visual Studio Code](https://code.visualstudio.com) with the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp).
 
-We will be using `yo aspnet` to generate the **Web Application Basic** template, you may follow the full instructions in [Building Projects with Yeoman](../client-side/yeoman.md) to create an ASP.NET Core project which show an **Empty Web** for reference.
+## Scaffolding applications using dotnet new
 
-Install the necessary yeoman generators and bower using npm.
+We will be using `dotnet new` to generate a new web application using the "Empty Web Template". Create a working directory for your project called _firstmacapp_ and change the directory to the newly created folder.
```

Comment:
Create a working directory for your project called *firstmacapp* . `cd` to *firstmacapp*.
Or whatever it is on the MAC. Try to keep sentences short.